src/test/regress/output/multi_agg_distinct.source

```diff
@@ -73,12 +73,14 @@ SELECT p_partkey, count(distinct l_orderkey) FROM lineitem_range, part
 (10 rows)
 
 RESET citus.large_table_shard_count;
--- Check that we don't support count(distinct) on non-partition column, and
--- complex expressions.
+-- Check that we support count(distinct) on non-partition column.
 SELECT count(distinct l_partkey) FROM lineitem_range;
-ERROR:  cannot compute aggregate (distinct)
-DETAIL:  table partitioning is unsuitable for aggregate (distinct)
-HINT:  You can load the hll extension from contrib packages and enable distinct approximations.
+ count 
+-------
+ 11661
+(1 row)
+
+-- Check that we don't support complex expressions.
 SELECT count(distinct (l_orderkey + 1)) FROM lineitem_range;
```

Comment:
We can leave this out of the scope, but I'm just curious about why we can't do this. Isn't this similar to what we did for distinct support on non-partition columns?

Reply:
we should be able to allow that if needed in the future.
It requires evaluation expression inside distinct clause. Once we reliably do that, we could also allow more complex expressions like ```case ... when ...``` and even function calls.

I haven't check the case with approximate distinct using hll. This single column restriction might be coming from there.